test/gpdb/Dockerfile

```diff
@@ -0,0 +1,4 @@
+ARG GPDB_IMAGE=gpdb6_regress:latest
```

Comment:
It looks defailt values don't work well.
Simple `bash run_docker.sh` doesn't allow to launch the test with the default image. Though, I guess there was such an intention.
Issues I faced:
1. at first, `bash run_docker.sh` will set $GPDB_IMAGE to blank value. Why not just omit the `--build-arg` if the argument is not passed to `run_docker.sh`?
2. after w/a item#1, docker tries to load the image from docker.io/library/gpdb6_regress:latest and fails. Why not to make like in the gprestore dockerfile:
```
ARG GPDB_IMAGE=gpdb6_regress:latest
FROM hub.adsw.io/library/$GPDB_IMAGE
```
?

Reply:
Have fixed. If you will have gpdb6_regress:latest in your own docker (locally for example) the default call will work. I won't use `hub.adsw.io` in test scripts.

Reply:
> I won't use hub.adsw.io in test scripts.

Why not? BTW, shouldn't we place all docker related testing stuff under `arenadata` as it is done in gpdb and gpbackup?

Reply:
> shouldn't we place all docker related testing stuff under arenadata
I'll ask. Initially i followed the logic that repo is public and somebody not from arendata may launch the script. I dont want to make bindigns to internal infrastructure things

Reply:
Have discussed this aspect. Have moved everything to arenadata

Reply:
Please update path in the description too.